v3/src/components/graph/components/axis.tsx

```diff
@@ -7,27 +7,26 @@ import {useDataSetContext} from "../../../hooks/use-data-set-context"
 import {getDragAttributeId, IDropData} from "../../../hooks/use-drag-drop"
 import {useInstanceIdContext} from "../../../hooks/use-instance-id-context"
 import {useAxis} from "../hooks/use-axis"
-import {AxisPlace, INumericAxisModel} from "../models/axis-model"
+import {AxisPlace, IAxisModel, INumericAxisModel} from "../models/axis-model"
 import {useGraphLayoutContext} from "../models/graph-layout"
 import {AxisDragRects} from "./axis-drag-rects"
-import {IGraphModel} from "../models/graph-model"
 
 import "./axis.scss"
 
 interface IProps {
-  graphModel: IGraphModel
+  getAxisModel: any // todo: Figure out how to declare this properly
```

Comment:
I was thinking it could just be `() => IAxisModel` and then clients could pass something like `() => graphModel.getAxisModel(place)` and then `place` wouldn't need to be passed explicitly, either.